helix-view/src/view.rs

```diff
@@ -60,6 +64,8 @@ impl JumpList {
     }
 }
 
+const GUTTERS: &[(Gutter, usize)] = &[(gutter::diagnostic, 1), (gutter::line_number, 5)];
```

Comment:
I think we need to somehow expose this in config, not sure if we want to do it in this patch or not.

Reply:
The width is currently there so that gutters have a constant width. We will probably make them more dynamic in the future.